onnx/backend/test/runner/__init__.py

```diff
@@ -163,8 +163,8 @@ def tests(self):  # type: () -> Type[unittest.TestCase]
                 setattr(tests, name, item.func)
         return tests
 
-    @staticmethod
-    def _assert_similar_outputs(ref_outputs, outputs, rtol, atol):  # type: (Sequence[Any], Sequence[Any], float, float) -> None
+    @classmethod
+    def _assert_similar_outputs(cls, ref_outputs, outputs, rtol, atol):  # type: (Sequence[Any], Sequence[Any], float, float) -> None
```

Comment:
since we already change this to classmethod, why not also turning it to public method?